lib/streamlit/elements/selectbox.py

```diff
@@ -45,7 +84,7 @@ def selectbox(
         kwargs: Optional[WidgetKwargs] = None,
         *,  # keyword-only arguments:
         disabled: bool = False,
-    ) -> Any:
+    ) -> Optional[Any]:
```

Comment:
Could this (along with the other `Optional[Any]`s) just be `Any`?

Although I don't have a strong opinion on this either way since there is merit to having these be consistent with the `Optional[T]` above.

Reply:
For return types, the benefit of spelling out all of the types you know you can encounter, in addition to `Any`, is that the developer has to deal with these specifically. This is different from parameter types where, indeed, `Optional[Any]` is equivalent to `Any`.

Here's a simplified example that illustrates this point: https://mypy-play.net/?mypy=latest&python=3.10&gist=8afc339c79f6377558e19a4f3c6491d3